src/header.ts

```diff
@@ -116,6 +133,20 @@ export interface HeaderConfig {
    * __Default value:__ `0`, indicating no limit
    */
   labelLimit?: number;
+
+  /**
+   * The orthogonal offset in pixels by which to displace the title from its position along the edge of the chart.
+   *
+   * __Default value:__ `10`, indicating no limit
+   */
+  labelOffset?: number;
+
+  /**
+   * Label orientation (`"top"`, `"bottom"`, `"left"`, or `"right"`).
+   *
+   * __Default value:__ `"left"` for rows and `"top"` for columns.
+   */
+  labelOrient?: TitleOrient;
```

Comment:
Does mapping label orient to orient of underlying title work?

I thin you have to do much more (correctly applying to header/footer, etc.)?